athina/__init__.py

```diff
@@ -1,6 +1,16 @@
 # __init__.py
-
 from .guard.guard import guard
 from .guard.exception import AthinaGuardException
 from . import evals
 from . import keys
+from .runner.run import EvalRunner
+
+def run(evals, data):
+    """
+    A convenience wrapper to run evaluation suites.
+
+    :param evals: A list of evaluations to be run.
+    :param data: The dataset over which evaluations are run.
+    """
+    # Call the EvalRunner's run_suite method directly
+    return EvalRunner.run_suite(evals=evals, data=data)
```

Comment:
I think we should put this in another file, and just have the imports in the __init__ file

Reply:
Moved it